File: Data Structures & Algorithms/evaluate-reverse-polish-notation/submission-1.py
class Solution:
    def evalRPN(self, tokens: List[str]) -> int:
        numbers = []

        for t in tokens:
            if t == "+" or t == "-" or t == "*" or t == "/":
                first = numbers.pop()
                second = numbers.pop()

                if t == "+":
                    numbers.append(first + second)
                elif t == "-":
                    numbers.append(second - first)
                elif t == "*":
                    numbers.append(first * second)
                elif t == "/":
                    numbers.append(int(second / first))
            else:
                numbers.append(int(t))
        

        return numbers[0]

        

File: Data Structures & Algorithms/two-integer-sum-ii/submission-1.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& numbers, int target) {
        int i = 0;
        int j = numbers.size() - 1;

        while (i < j){
            if (numbers.at(i) + numbers.at(j) > target){
                j--;
            }else if (numbers.at(i) + numbers.at(j) < target){
                i++;
            }else{
                return vector<int>{i+1, j+1};
            }
        }

        return vector<int>{};
    }
};
